library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -149,7 +149,7 @@ public void onCreate() {
                 Downloads.Impl.ALL_DOWNLOADS_CONTENT_URI,
                 true, mObserver);
 
-        mExecutor = new DownloadExecutorFactory().createExecutor(this);
+        mExecutor = DownloadExecutorFactory.newInstance(this).createExecutor();
```

Comment:
Went back to a different newInstance method: https://github.com/novoda/download-manager/commit/f5b24f1fb27db739e5b5e84d54d64c9bbe16384f